Makefile

```diff
@@ -23,7 +23,7 @@ $(EMBEDDED_STATIC): static/kubeapps-objs.yaml
 	$(GO) generate
 
 static/kubeapps-objs.yaml:
-	KUBEAPPS_VERSION=$$(git describe --tags --abbrev=0) ;\
+	KUBEAPPS_VERSION=$${VERSION:-latest} ;\
```

Comment:
wouldn't tagname `dev` be more appropriate here?

Reply:
`latest` has the special case in Kubernetes that makes the imagePullPolicy Always, though I do see your point... Given the binaries won't ever point to latest I think it's okay to have latest be broken though.